src/main/java/io/appform/dropwizard/actors/connectivity/ConnectionConfig.java

```diff
@@ -0,0 +1,27 @@
+package io.appform.dropwizard.actors.connectivity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ConnectionConfig {
+
+    @NotNull
+    @NotEmpty
+    private String name;
+
+    @Min(1)
+    @Max(100)
```

Comment:
Max seems a bit low.

Reply:
This is per connection. As per RMQ recommendations, this should ideally be number of cores * 2 for the JVM process. Should I increase even further ?

Reply:
Increased for now.